markhor_flippers/src/markhor_hw_interface_flippers.cpp

```diff
@@ -232,7 +254,7 @@ void MarkhorHWInterfaceFlippers::write()
   be above or under the limit of the flipper.
  */
 
-  printDriveInfo(front_left_drive_);
+ // printDriveInfo(front_left_drive_);
```

Comment:
Forgotten comment?

Reply:
For me i think it is more usable having only one drive loging in the ros info but we can restore it to full loging if you'd like

Reply:
I don't mind one or the other. However, I don't like this in-between with 3 commented out lines.